sklearn/model_selection/tests/common.py

```diff
@@ -0,0 +1,23 @@
+"""
+Common utilities for testing model selection.
+"""
+
+import numpy as np
+
+from sklearn.model_selection import KFold
+
+
+class CustomSplitter:
+    """A wrapper to make KFold single entry cv iterator"""
+    def __init__(self, n_splits=4, n_samples=99):
+        self.n_splits = n_splits
+        self.n_samples = n_samples
+        self.indices = KFold(n_splits=n_splits).split(np.ones(n_samples))
```

Comment:
if split returns a list, then the premise of this test is false. use `iter()` to make sure it's not reentrant.